home/admin.py

```diff
@@ -0,0 +1,28 @@
+from django.contrib.admin import ModelAdmin
+from django.contrib import admin
+from django.utils.html import format_html, format_html_join
+
+from home.models import Student
+
+
+class StudentAdmin(ModelAdmin):
+    """This class displays records of the project that needs to
```

Comment:
`This class`, `this function`, `this method` ...
its unneeded, if it class just explain why they need, if method/function just explain what they do